.gitignore

```diff
@@ -25,4 +25,5 @@ bin
 vendor/
 config/default/manager_image_patch.yaml-e
 .vscode/
-cover.html
\ No newline at end of file
+cover.html
+upgrade-manager
```

Comment:
what is this ignoring?

Reply:
cover.html - is the generated code coverage file.
upgrade-manager - binary if you run `go build`.

Reply:
Hmm.. technically we should use makefile `make manager` which places binaries in /bin/manager, which is already ignored.
Does it make sense to use that instead? ignoring everything named `upgrade-manager` might be problematic?

Reply:
It is possible to set full path like /cover.html /upgrade-manager.

Reply:
Anyway, I can revert this change completely, this is not a goal of this PR.

Reply:
Yep, cover.html can stay (it was already there), but I think we should remove that one (`upgrade-manager`) for now as it overrides the makefile behavior.
Otherwise this PR looks good.